inst/config/note.json

```diff
@@ -30,6 +30,11 @@
       "index_to_check": 1,
       "text_to_check": "The longest 5 functions are"
     },
+    {
+      "length": 1,
+      "index_to_check": 1,
+      "text_to_check": "Consider adding runnable examples to man pages"
```

Comment:
I would not add it as exception because again before the release we will need to add all the required examples :/

Reply:
In my opinion we could add it for the time being and reevalute it at the end of Q2, with other notes.

Reply:
I've identified gDRutils as the only package with some examples missing. @bczech will fix it with https://github.com/gdrplatform/gDRutils/pull/67/commits/2ee2fa8dca6b3a35321daabf2d16c78288fae9d0. Thus we can remove this exception.